implementations/rust/ockam/ockam_command/src/node/show.rs

```diff
@@ -52,10 +52,54 @@ async fn run_impl(
     ctx: Context,
     (opts, cmd): (CommandGlobalOpts, ShowCommand),
 ) -> miette::Result<()> {
-    let node_name = get_node_name(&opts.state, &cmd.node_name);
+    if !opts.terminal.can_ask_for_user_input() {
+        show_node(&opts, &ctx, &get_node_name(&opts.state, &cmd.node_name)).await?;
+        return Ok(());
+    }
+
+    let node_names = opts.state.nodes.list_items_names()?;
+    match node_names.len() {
+        0 => {
+            opts.terminal
+                .stdout()
+                .plain("There are no nodes to show")
+                .write_line()?;
+        }
+        1 => {
+            show_node(&opts, &ctx, &node_names[0]).await?;
+        }
+        _ => {
+            let selected_node_names = opts.terminal.select_multiple(
+                "Select one or more nodes that you want to show".to_string(),
+                node_names,
+            );
+
+            if selected_node_names.is_empty() {
+                opts.terminal
+                    .stdout()
+                    .plain("No nodes selected to show")
+                    .write_line()?;
+                return Ok(());
+            }
+
+            if opts.terminal.confirm_interactively(format!(
```

Comment:
I've been testing the command manually and I think this check here should be removed. In this case we are just showing data, it's not like in the `node delete` command where we want the user to confirm the action.

Reply:
I'm good with this. It's the user flow described in https://github.com/build-trust/ockam/issues/6471 by @nazmulidris 
I'll create a commit first.

Reply:
commit [e029ade](https://github.com/build-trust/ockam/pull/6665/commits/e029adef4b45f8961acba268e2cefae018c4317d).

Reply:
@adrianbenavides I think removing the confirmation prompt might make make more sense in this case, since the operation is non destructive. In the case where the operation is destructive it does make sense.

I can update the existing issues to reflect this change in user flow that @mickm3n was referring to (from the issues).

> - I currently have 2 variants of tuify issues. One where in the final step (after getting user confirmation) a checkbox is displayed showing whether the operation succeeded or not. And another where no checkbox is displayed.
> - I will add another variant, where the operation that is performed is non-destructive, so there might not be any need to get confirmation from the user before proceeding.